libs/tree-sitter-wing/grammar.js

```diff
@@ -13,9 +13,10 @@ const PREC = {
   UNARY: 120,
   OPTIONAL_TEST: 130,
   POWER: 140,
-  MEMBER: 150,
-  CALL: 160,
-  OPTIONAL_UNWRAP: 170,
+  SUBSCRIPT: 150, // x[0]
```

Comment:
subscript? This could match the rule instead with something like STRUCTURED_ACCESS